research-software-directory/RSD-as-a-service
# Plpgsql to sql

## Change database functions

Changes proposed in this pull request:

* Change database functions from `plpgsql` to `sql` to improve performance
* Reorder the functions, because at creation time it is checked if their dependencies exist

How to test:

* `docker compose down --volumes && docker compose build --parallel && docker compose up --scale data-generation=1`
* Browse the RSD (http://localhost), check if everything works

PR Checklist:
